HelloWorld/configure_ios_test_app.ts

```diff
@@ -14,7 +14,7 @@ exports.run = function () {
 
   childProcess
     .execSync(
-      `xcodebuild build CODE_SIGNING_ALLOWED=NO -project ${expectedProjectPath} -configuration Debug -sdk iphonesimulator`,
+      `xcodebuild build -project ${expectedProjectPath} -configuration Debug -sdk iphonesimulator`,
```

Comment:
By explicitly having `CODE_SIGNING_ALLOWED=NO` when compiling the app code, the generated binaries would not work with Appium (i.e. Appium would not be able to get a context to the webview in the app). Removing this solves that problem.